.eslintignore

```diff
@@ -1,4 +1,3 @@
-assets/js/jquery/3.5.1/jquery.min.js
 _site
```

Comment:
I don't recall deleting this line. Wondering if a script removed it. Looking into this